### ISSUE TITLE
Include complete metadata in .json files

### ISSUE THREAD

**Issue by sschmaus** [kmgill/mars-raw-utils#10]
Since the update to metadata handling there are many values now missing from the JSON metadata files. 

My workflow for photogrammetry and subsequent animations depends on this information, for example the sclk timer, the CAHVOR camera model and position information. 

I would be really helpful to get these values back into the JSON file, preferably just in the old structure or otherwise as a sub-array, so the general structure is still mostly the same.

**Comment by kmgill** [comment]
Yes, I'll be working the metadata values back into it. I'm unifying the metadata across what Mars2020, MSL, and InSight web services provide.

**Comment by kmgill** [comment]
It's good to know you need the data back, so I'll give it a higher priority.

**Comment by sschmaus** [comment]
Thank you. 
Its not super urgent, I downgraded a few commits, so for it's working again for me now. 

Only when I want to use the any of the new features you are working on, this would really be needed.

**Comment by kmgill** [comment]
Got the stuff in there.

**Comment by sschmaus** [comment]
Sorry for opening this topic up again, but I noticed a few data fields that would make sense to be included as well. Namely these are `xyz`, `date_received` and maybe also `dimension` and `sampletype`. 
The rest is mostly browse related links that shouldn't matter when working with the images. 
From my position there is no time pressure to include them though, my current needs are all covered.

**Comment by kmgill** [comment]
Reopening

**Comment by kmgill** [comment]
Added the four recommended fields, as availability allowed.

**Comment by sschmaus** [comment]
While working on parsing the rover camera model data I noticed that `attitude` is still missing from the MRU metadata (this should be the last one missing). This specifies the rover orientation as a quaternion and is kind of important if you want to know how the rover and in result the cameras, are oriented. 
It's available on both MSL and M20.

**Comment by kmgill** [comment]
Added `attitude` field to json metadata files

**Comment by sschmaus** [comment]
Thank you! I think the issue is resolved now. 

In the long run it would be neat to have all the number values be formatted as numbers instead of strings, but this is not a high priority for me. 

Are you planning any changes to the metadata system? I think you mentioned that once.
